Gruntfile.js

```diff
@@ -31,8 +31,7 @@ module.exports = function(grunt) {
 					"css/main.css",
 					"modules/default/calendar/calendar.css",
 					"modules/default/clock/clock_styles.css",
-					"modules/default/currentweather/currentweather.css",
-					"modules/default/weatherforcast/weatherforcast.css"
+					"modules/default/**/*.css"
```

Comment:
This change is not part the request to introduce in this PR, you should be move in another PR.

Reply:
This was an incidental change when adding `newsfeed` CSS to `stylelint` ... wasn't clear why some module CSS shouldn't lint, except that it's more of a dependency than source file. But if I'm reverting CSS addition, so will remove this as well.